src/theme/index.hbs

```diff
@@ -224,20 +224,13 @@
 
         {{#if google_analytics}}
         <!-- Google Analytics Tag -->
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-109503488-17"></script>
```

Comment:
It has been a long time since I have used Google analytics, but isn't `UA-109503488-17` a personal id that should not be here?

Reply:
Yup, dunno how I missed that, will replace it (script does replace it)